client/src/extension.ts

```diff
@@ -35,22 +20,6 @@ const LANGUAGES = [
   "javascriptreact",
 ];
 
-interface TsLanguageFeatures {
-  getAPI(version: 0): TsLanguageFeaturesApiV0 | undefined;
-}
-
-async function getTsApi(): Promise<TsLanguageFeaturesApiV0> {
-  const extension: vscode.Extension<TsLanguageFeatures> | undefined = vscode
-    .extensions.getExtension(TS_LANGUAGE_FEATURES_EXTENSION);
-  const errorMessage =
-    "The Deno extension cannot load the built in TypeScript Language Features. Please try restarting Visual Studio Code.";
-  assert(extension, errorMessage);
-  const languageFeatures = await extension.activate();
-  const api = languageFeatures.getAPI(0);
-  assert(api, errorMessage);
-  return api;
-}
```

Comment:
Extracted out to its own file and burried some complexity—single `refresh()` method now.

Reply:
👍